### DISCUSSION TITLE
error with rasterize_terrain on LAScatalog

### DISCUSSION THREAD

**Discussion by JonBatch** [r-lidar/lidR#560] [General]
Hello, when running the following code:

```r
ctg<-readLAScatalog("datasetsA/king_county_2016/laz/")
TILEdtm <- rasterize_terrain(ctg,res=30,algorithm = tin())
```

on lidar tiles downloaded from the Washington DNR lidar portal (https://lidarportal.dnr.wa.gov/) I am getting this error:

```r
ctg<-readLAScatalog("datasetsA/king_county_2016/laz/")
#> Be careful, some tiles seem to overlap each other. lidR may return incorrect outputs with edge artifacts when processing this catalog.
> TILEdtm <- rasterize_terrain(ctg,res=30,algorithm = tin())
Processing [==================================>-----------------------------------]  50% (1/2) eta: 43s
#> Error in parse(text = x@definition) : <text>:1:1405: unexpected symbol
#> 1: ",north,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["Engineering survey, topographic mapping."],AREA["United States (USA) - Washington - counties of Adams; Asotin; Benton; Clark; Columbia; Cow
```

I had been using `grid_terrain` and I am trying to update workflows per the suggestions in the lidR documentation however, `grid_terrain` works and I can't seem to get `rasterize_terrain` to work on a `LAScatalog`. Am I missing some additional input required for `rasterize_terrain`?                                                                                        

I have tried adjacent tiles from 3 different acquisitions on two different machines and I have gotten the same error. I have run this code in the past (using 'grid_terrain') with no issues at all. I assume that perhaps with the recent updates to lidR that something has changed? I also assume that the error is related to user-defined information included in the laz header. To reproduce the error you should be able to download any two adjacent tiles from the 2016 king county lidar aquistion over Seattle.

**Comment by Jean-Romain** [comment]
Without reproducing I'd say that your files have invalid CRS that were parsed anyway when CRS were handled with `sp` but now CRS are handled with `sf` it fails at loading the file.

Please share a link a problematic file (not to shaded DTMs) so I can investigate futher.

**Comment by JonBatch** [comment]
This link will hopefully allow you to download two of the tiles from the 2016 King County lidar acquisition. 

https://lidarportal.dnr.wa.gov/download?geojson=%7B%22type%22%3A%22Polygon%22%2C%22coordinates%22%3A%5B%5B%5B-122.3035%2C47.6436%5D%2C%5B-122.3035%2C47.6567%5D%2C%5B-122.3028%2C47.6567%5D%2C%5B-122.3028%2C47.6436%5D%2C%5B-122.3035%2C47.6436%5D%5D%5D%7D&ids=836

**Comment by Jean-Romain** [comment]
Bug confirmed #561
